README.md

```diff
@@ -329,6 +329,17 @@ has exhausted its retries you can add the `RetryableJobsFilter`:
 Airbrake.add_filter(Airbrake::Sidekiq::RetryableJobsFilter.new)
 ```
 
+To ignore certain classes `x` times use the `IgnorableClassFilter`
+
+```ruby
+Airbrake.add_filter(
+  Airbrake::Sidekiq::IgnorableClassFilter.new(
+    ignorable_classes: ['SomeIgnorableClass'],
```

Comment:
Is the class here a worker name or an error class?

Reply:
This is supposed to be an error class. Ill renamed this to be `ignorable_error_classes`

Reply:
What do you think about `ignorable_errors`? 

```
ignorable_errors: %w[ArgumentError RuntimeError]
```

It looks more terse and seems to convey the meaning, too.